tokio/src/time/driver/mod.rs

```diff
@@ -333,28 +334,33 @@ impl Inner {
         self.elapsed.load(SeqCst)
     }
 
+    #[cfg(test)]
+    #[cfg(loom)]
+    fn num(&self, ordering: ::std::sync::atomic::Ordering) -> usize {
```

Comment:
Extremely minor nit: I think this style of import (prefixed with `::`) is considered unidiomatic in Rust 2018.

Reply:
Hrm. Yeah, I don't love it. This change was introduced in d7f07b9  to avoid the build error present in fa33129. I'm very open to other alternatives. It does look bad.

Reply:
FWIW, I am pretty sure that this _could_ just be
```suggestion
    fn num(&self, ordering: std::sync::atomic::Ordering) -> usize {
```
and then you'd avoid the unused import error. Alternatively you could have a separate import of `Ordering` with `#[cfg(all(test, loom))]` on it...